WireAPI/Sources/WireAPI/APIs/AccountsAPI/Requests/UpgradeToTeamRequestBodyV7.swift

```diff
@@ -0,0 +1,38 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+struct UpgradeToTeamRequestBodyV7: Codable, Sendable {
+    let currency: String?
+    let icon: String
+    let icon_key: String?
+    let name: String
+
+    init(
+        currency: String? = nil,
```

Comment:
Out of interest, do you expect this to be called with default values?

Reply:
Those are optional parameters that the backend endpoint _can_ accept, but they are optional.
The only value we get during the UI flow being the team's name, and nothing else, we don't actually _have_ anything to provide there.

So I personally would not even put those parameters in the `init`, but then our request body doesn't match the endpoint's anymore.

What would you suggest?

Reply:
I think it is fine. I personally don't like it when we add default parameters to init methods but the users of the init never rely on the default values as sometimes this means an extra test without benefit. But I think it makes sense in this case as we will rely on the default values.